ansible_files/vars/ci_equinix_infrastucture.yml

```diff
@@ -0,0 +1,7 @@
+# Equinix related vars, built from CI vars
+equinix_project_id: "{{ lookup('file', cluster_profile_dir + '/packet-project-id') | trim }}"
+equinix_auth_token: "{{ lookup('file', cluster_profile_dir + '/packet-auth-token') | trim }}"
+equinix_ssh_private_key_path: "{{ cluster_profile_dir }}/packet-ssh-key"
```

Comment:
I'm a bit unsure about this file belonging here in test-infra. I think things like ``packet-ssh-key`` that are configured once in ``openshift/release`` should be mentioned only there

Either way, it's not a blocker. Just wanted to add a note about it

Reply:
If needed, they can be overridden from the release config using the `ANSIBLE_EXTRA_VARS` environment variable: https://github.com/openshift/release/blob/master/ci-operator/step-registry/assisted/equinix/setup/assisted-equinix-setup-ref.yaml#L19-L21

It should help if we want to make consistent changes in the `release` repo without changing this one.